htlcswitch/test_utils.go

```diff
@@ -495,9 +493,41 @@ func (n *threeHopNetwork) stop() {
 	for i := 0; i < 3; i++ {
 		<-done
 	}
+}
+
+// createClusterChannels creates lightning channels which are needed for
```

Comment:
Is this used anywhere in this diff? If so, then it can safely be removed. 

Why is this method preferred over the prior `newThreeHop` network function?